src/markdown/downloads/disbursements-by-month.md

```diff
@@ -24,34 +24,91 @@ Download fiscal year data:
 
 _Month_ The month the disbursement occurred
 
-_Year_ The year the disbursement occurred
+_Calendar Year_ The period between January 1 and December 31 for a given year.
 
 _Fund Type_ The type of recipient receiving the disbursement
 
+* _BLM-Nat Petrol Reserve-2_ Funds from oil and gas leases directed to the National Petroleum Reserve in Alaska.
+* _Bureau of Land Management_ Certain funds are directed back to the federal agencies that administer these lands and waters to help cover the agencies’ operational costs.
+* _Bureau of Ocean Energy Management_ Certain funds are directed back to the federal agencies that administer these lands and waters to help cover the agencies’ operational costs.
+* _Bureau of Safety & Environmental Enforcement_ Certain funds are directed back to the federal agencies that administer these lands and waters to help cover the agencies’ operational costs.
+* _Fish & Wildlife_ Certain funds are directed back to the federal agencies that administer these lands and waters to help cover the agencies’ operational costs.
+* _Forest Service_ Certain funds are directed back to the federal agencies that administer these lands and waters to help cover the agencies’ operational costs.
+* _Land & Water Conservation Fund_ This fund provides matching grants to states and local governments to buy and develop public outdoor recreation areas across the 50 states. The Gulf of Mexico Energy Security Act (GOMESA) of 2006 specifies that 12.5% of revenues from certain gulf leases be directed to the LWCF stateside program. States can receive up to $125 million a year in funding. This portion of LWCF funds is not subject to the congressional appropriation process, but are otherwise treated similarly to regular LWCF funds.
+* _Lease Process Improvement (BLM)_ Permit Processing Fund, Bureau of Land Management (BLM), was established by the Department of the Interior as authorized by 119 Stats. 725, 671, and 695, for the deposit of mineral lease rentals. The Department of the Interior is authorized to use funds for the coordination and processing of oil and gas use authorizations on Federal land onshore, under the jurisdiction of the Pilot Project offices.
+* _Native American Tribes & Allottees_ ONRR disburses 100% of revenue collected from resource extraction on Native American lands back to Native American  tribes, nations, and individual Indian landowners.
+* _Reclamation Fund_ Established by Congress in 1902, funds the development of irrigation projects on arid and semiarid lands of the 17 western states.
+* _State Fund_ disbursed to states fall under the jurisdiction of each state, and each state determines how the funds will be used.
```

Comment:
Use consistent format. For the others, the explanation starts with a capitalized letter. Also, looks like this State Fund entry looks like it might be incomplete or truncated?